pkg/election/election.go

```diff
@@ -80,6 +101,32 @@ func NewElection(ctx context.Context, cli *clientv3.Client, sessionTTL int, key,
 		return nil, terror.ErrElectionCampaignFail.Delegate(err, "create the initial session")
 	}
 
+	for {
```

Comment:
Block launching until one leader elected, right?

Reply:
yes, because some other action in the init phase depend on the gRPC api such as `ShowDDLLocks`, so to make things easy, we try to determine a leader first. As it's embeded etcd, this would be quite fast